server/home/home-web/src/main/java/io/holoinsight/server/home/web/controller/MarketplacePluginFacadeImpl.java

```diff
@@ -522,6 +568,9 @@ public void doManage() {
         if (null != ms && !StringUtils.isEmpty(ms.tenant)) {
           customPluginRequest.getTarget().setTenant(ms.tenant);
         }
+        if (null != ms && !StringUtils.isEmpty(ms.getWorkspace())) {
```

Comment:
Should we use StringUtils.isEmpty or StringUtils.isBlank for workspace? The code needs to be consistent.